### ISSUE TITLE
Align DSR Split View to SLDS

### ISSUE THREAD

**Issue by kwilloughby-sf** [salesforce/design-system-react#2131]
- Do not change the information on the right unless the user clicks on an item or presses enter
- The user should be able to navigate the items using the up/down arrow keys which should move a 1px blue border up/down
- The selected item should only have a 1px blue border if the use is focused on that item
- If the user is focused on another item the selected item should be indicated by the 4px blue box shadow

**Comment by kwilloughby-sf** [comment]
@jcjuett

**Comment by jcjuett** [comment]
@kwilloughby-sf just so I understand--we are using custom focus boxes and not relying on user agent focus?

**Comment by kwilloughby-sf** [comment]
@jcjuett, we're using the default SLDS styling which uses custom focus boxes.

**Comment by interactivellama** [comment]
@kwilloughby-sf The following ask is a good interaction pattern but is dependent on how the consumer sets up their app. In summary, we can't guarantee this UX interaction and therefore can not back it into the component UI.

 [Accessibility] Add instructions to the first item in the list to indicate Shift + Tab will take the user back to the list after selecting an item.

**Comment by interactivellama** [comment]
Moved interaction patterns to issue #2310

**Comment by kwilloughby-sf** [comment]
@interactivellama, regarding "[Accessibility] Add instructions to the first item in the list to indicate Shift + Tab will take the user back to the list after selecting an item"...

Is the concern that the consumer might not move the focus to the right side content, therefore, Shift + tab might not go back to the correct thing?

CC @jcjuett

**Comment by interactivellama** [comment]
yes, that issue and most likely it would be multiple shift tabs needed to return to SplitView and the SplitView could be at the bottom of the DOM so it could be Tab forward instead to. We can't control it.
